lib/peep.rb

```diff
@@ -0,0 +1,33 @@
+require 'pg'
+
+class Peep
+    attr_reader :id, :message
+
+    def initialize(id:, message:)
+        @id = id
+        @message = message
+    end
+
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
+            connection = PG.connect(dbname: 'chitter_test')
+        else
+            connection = PG.connect(dbname: 'chitter')
+        end
```

Comment:
If you observe these lines and lines 23-27, these are repeated. Can all of this be simplified somehow in this Class to follow the DRY principle?